.github/ISSUE_TEMPLATE/DOCUMENTATION.md

```diff
@@ -0,0 +1,51 @@
+---
+name: "📜📖📄📘 📚  Documentation enhancement"
+about: Suggest edits for the Wagtail documentation
+title: ''
+labels: 'Documentation'
+assignees: ''
+
+---
+
+<!--
+  Summarise the documentation change you’re suggesting in the Issue title.
+-->
+
+### Pertinent section of the Wagtail docs 
+
+<!--
+  Copy the section link here.
+-->
+
+(Insert the permalink of the section heading in the Wagtail docs. Choose the most specific section / lowest-level heading to which your suggested change would apply. (In the docs, hover over the the heading to see the permalink icon.))
+
+### Suggestion type
+
+<!--
+  Select one of the following options by placing an 'x' inside the corresponding brackets, or just delete the other options. 
+-->
+
+|  x  | Type of change  | Fix/enhancement |
+| --- | ------------- | ------------- |
+| [ ] | Copy edit | Fix spelling or grammar errors, improve clarity
+| [ ] | Content correction | Correct inaccurate explanations, instructions, or code 
+| [ ] | New content | Add a new topic or provide more detailed instructions or examples 
+| [ ] | Reorganization | Move topic(s) 
+| [ ] | Other suggestion | Anything else
+
+### Details
+
+<!--
+  Provide a clear and concise description of what you want to happen.
+-->
+
+(Write your answer here.)
+
+### Proposed change (for brief corrections/revisions)
+
+<!--
+  - If you're suggesting a correction or revision of existing passage, write the corrected/revised passage here.
+?  -  If you want to contribute new or reorganized content please create a pull request...
+-->
+
+(Write your answer here.)
```

Comment:
Even for documentation changes, I think we should encourage people to use pull requests only. They are much easier for us to manage than code snippets in issues. I would suggest removing this section.

```suggestion

<!--
  If you're suggesting a very specific change to the documentation, feel free to directly submit a pull request.
-->
```